manual_qa_tests/workflows/succeed_with_snapshots_disabled.py

```diff
@@ -0,0 +1,47 @@
+import aqueduct as aq
+
+NAME = "succeed_with_snapshots_disabled"
+DESCRIPTION = """* Workflows Page: should succeed.
+* Workflow Details Page: everything should succeed except for 'check_disabled_artf' and op_disabled_artf'."""
+
+
+@aq.op(requirements=[])
+def op_disabled(df):
+    return df
+
+
+@aq.metric(requirements=[])
+def metric(df):
+    return df.shape[0]
+
+
+@aq.check(requirements=[])
+def check(count):
+    return count > 10
+
+
+@aq.check(requirements=[])
+def check_disabled(count):
+    return count < 10
+
+
+def deploy(client, resource_name):
```

Comment:
@vsreekanti would like your input to confirm this call chain should result in:
* reviews, op_artf and check_disabled_artf to be deleted
* outputs of `metrics` and `check` are persisted

Reply:
Actually I feel allowing disabling check artf is very confusing. I'm wondering if we should just return an error if the user attempts to do so. Maybe the same applies to params.

Disabling metric artf makes sense to me though, since esp. table metrics can have sensitive content as they are just a grammar sugar over regular operators.